Calculator/Calculator/Model/Formula.swift

```diff
@@ -26,7 +25,7 @@ struct Formula {
             }
             let calculatedValue = `operator`[0].calculate(lhs: reuslt, rhs: partialResult)
```

Comment:
```suggestion
            let calculatedValue = operators.first?.calculate(lhs: reuslt, rhs: partialResult)
```
=> 배열이니깐 operators가 낫지 않을까요~?